javaslang/src/main/java/javaslang/Lazy.java

```diff
@@ -133,12 +135,13 @@ private Lazy(Supplier<? extends T> supplier) {
      */
     @Override
     public T get() {
+        // using a local var speeds up the double-check idiom by 25%, see Effective Java, Item 71
```

Comment:
this comment if for review only, I don't think it should stay in the code.
25%, based on what?